app/controllers/search_controller.rb

```diff
@@ -1,15 +1,16 @@
 class SearchController < ApplicationController
+  require 'faraday_middleware/aws_sigv4'
   respond_to :json
   before_action :ensure_json!
   before_action :authenticate_user!, except: 'ping'
 
   def search
-    client = Elasticsearch::Client.new log: false
+    client = configure_elasticsearch
```

Comment:
It looks to me like you are creating a new ES client for every request. I would think this is probably not what we want. I'm not sure how Rails typically handles this, but I would guess client creation might go in an initializer or something?

Reply:
Yup. You are correct. I'll adjust (stupid following docs instead of thinking about what is going on).